dev-python/python-language-server/python-language-server-0.32.0.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
```

Comment:
Can we add Python 3.8 support?

Reply:
Nope, sadly not, dependencies were missing py3_8 support last I checked